index.html

```diff
@@ -0,0 +1,122 @@
+          <p>a web developer</p>
+        </div>
+      </div>
+    </section>
+
+    <section id="work">
+      <section id="projects" ;>
+        <h2>These are some of my projects</h2>
+        <!-- First Tile  -->
+        <div class="project-tile-container">
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"
+              />
+            </div>
+
+            <div class="project-title">
+              <h3>Tribute Page</h3>
+            </div>
+          </div>
+
+          <!-- Second Tile  -->
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png"
+              />
+            </div>
+            <div class="project-title">
+              <h3>Random Qoute Machine</h3>
+            </div>
+          </div>
+
+          <!-- Third Tile  -->
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png"
+              />
+            </div>
+            <div class="project-title">
+             <h3>Javascript Calculator</h3>
+            </div>
+          </div>
+
+          <!-- fourth tile   -->
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"
+              />
+            </div>
+            <div class="project-title">
+              <h3>Map Data Across the Globe</h3>
+            </div>
+          </div>
+
+          <!--fifth Tile -->
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png"
+              />
+            </div>
+
+            <div class="project-title">
+              <h3>Wikipedia Viewer</h3>
+            </div>
+          </div>
+
+          <!-- sixth tile   -->
+          <div class="project-tile">
+            <div class="image-container">
+              <img
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png"
+              />
+            </div>
+            <div class="project-title">
+              <h3>Tic Tac Toe Game</h3>
+            </div>
+          </div>
+        </div>
+
+        <a
+          target="_blank"
+          id="profile-link"
+          href="https://github.com/martinkarugaba/Akinmola-Omotolani-and-Martin-Karugaba"
+        >
+          <span>Show All</span> 
+          <i class="fa-solid fa-angle-right icon"></i>
+        </a>
+      </section>
+    </section>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
```

Comment:
Hi @martinkarugaba, Good job :+1:  for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and provide for reference for suggested changes, But please kindly makes sure that you re-phrase the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change.